pkg/pillar/cmd/zedrouter/acl.go

```diff
@@ -34,6 +34,38 @@ const MAXACEID = 65535
 // MINACEID : IDs till 100 are reserverd for internal usage.
 const MINACEID = 101
 
+// Connection mark is used to remember for a given flow to which application it belongs
+// and which ACE was applied.
+// The 32 bits of a connmark are used as follows:
+//  +------------------------+---------------+------------------+
+//  | Application ID (8bits) | Action (1bit) | ACE ID (23 bits) |
+//  +------------------------+---------------+------------------+
+// where: Drop action = 1; Allow action = 0
+const (
+	appIDMask     = 0xff << 24
+	aceActionMask = 0x1 << 23
+	aceDropAction = aceActionMask
+	aceIDMask     = 0x7fffff
+	// By default, traffic not matched by any ACE is dropped.
+	// For this default rule we use the maximum integer value available for ACE ID.
+	defaultAceID = aceIDMask
```

Comment:
Should we name this dropAceID? It read a bit odd in the code which sets up the drop to see it use defaultAceID

Reply:
Or defaultDropAceID since we do drops for specific rules as well?

Reply:
I like `defaultDropAceID`, will change it to that.

Reply:
done